x-pack/plugins/ml/public/application/explorer/explorer_utils.js

```diff
@@ -544,7 +544,8 @@ export async function loadDataForCharts(
   latestMs,
   influencers = [],
   selectedCells,
-  influencersFilterQuery
+  influencersFilterQuery,
+  useRequestCount = true
```

Comment:
what does it mean?

Reply:
As mentioned in slack, this is because the original code has a `requestCount` on the file level, so whenever `loadDataForCharts` is called multiple times, it auto resolves/cancels previous calls. I think this is intentional for the Anomaly Explorer view, but since for embeddables there can be multiple explorer panels at the same time, this will cause unintentional behavior. I renamed this to `takeLatestOnly` for now but I think we can revisit this and refactor it a bit better in a follow up PR.